puppet/ha-puppet/screenshot.js

```diff
@@ -184,7 +169,57 @@ export class Browser {
     return this.page;
   }
 
-  async screenshotHomeAssistant({ pagePath, viewport, extraWait }) {
+  async initializeLocalStorage(page, hassUrl, clientId, darkMode) {
+    const hassLocalStorage = getHassLocalStorageDefaults(darkMode);
+
+    await page.evaluate(
+      (hassUrl, clientId, token, hassLocalStorage) => {
+        for (const [key, value] of Object.entries(hassLocalStorage)) {
+          localStorage.setItem(key, value);
+        }
+        localStorage.setItem(
+          "hassTokens",
+          JSON.stringify({
+            access_token: token,
+            token_type: "Bearer",
+            expires_in: 1800,
+            hassUrl,
+            clientId,
+            expires: 9999999999999,
+            refresh_token: "",
+          }),
+        );
+      },
+      hassUrl,
+      clientId,
+      this.token,
+      hassLocalStorage,
+    );
+  }
+
+  async updateTheme(darkMode) {
+    if (!this.page) return;
```

Comment:
This should either raise an error or just not be checked and let it crash. Let's make it a private method so it is fine to do so.